flowmachine/tests/test_labelled_spatial_aggregate.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+
+from flowmachine.core.dummy_query import DummyQuery
+from flowmachine.features import TotalLocationEvents
+from flowmachine.core import make_spatial_unit
+from flowmachine.features import SubscriberHandsetCharacteristic
+from flowmachine.features.location.labelled_spatial_aggregate import (
+    LabelledSpatialAggregate,
+)
+from flowmachine.features.subscriber.daily_location import locate_subscribers
+
+
```

Comment:
I think a couple of other tests would be good:
- One test that this works with multiple `label_columns`
- And a test that re-aggregating gives the expected result (i.e. summing counts over all labels gives the same result as the equivalent un-labelled SpatialAggregate query)